OrbitGl/TracepointDataView.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINTDATAVIEW_H
+#define ORBIT_TRACEPOINTDATAVIEW_H
+
+#include "DataView.h"
+#include "TracepointData.h"
+
+class TracepointsDataView : public DataView {
+ public:
+  TracepointsDataView();
+
+  const std::vector<Column>& GetColumns() override;
+  int GetDefaultSortingColumn() override { return kColumnName; }
+  std::vector<std::string> GetContextMenu(int clicked_index,
+                                          const std::vector<int>& selected_indices) override;
+  std::string GetValue(int row, int column) override;
+
+  void OnContextMenu(const std::string& action, int menu_index,
+                     const std::vector<int>& item_indices) override;
+  bool WantsDisplayColor() override { return true; }
+  bool GetDisplayColor(int row, int column, unsigned char& red, unsigned char& green,
+                       unsigned char& blue) override;
+  std::string GetLabel() override { return "Tracepoints"; }
+  bool HasRefreshButton() const override { return true; }
+  void OnRefreshButtonClicked() override;
+
+  void SetTracepoints(int32_t process_id, const std::vector<TracepointData*>& Tracepoints);
+
+ protected:
+  void DoSort() override;
+  void DoFilter() override;
+
+ private:
+  const TracepointData* GetTracepoint(uint32_t row) const;
+
+  std::vector<TracepointData*> Tracepoints_;
+
+  enum ColumnIndex {
+    kColumnName,
+  };
+};
```

Comment:
This class is not yet implemented? maybe extract this and TracepiontData class to a folloup PR?

It is not clear how loaded_ flag in TracepointData is used at this point, maybe you will need only proto class in UI implementation.